connectors/mcp_debug_tool.py

```
@@ -586,43 +649,47 @@
         }
     ]
 }
 
+
 # Usage Example
 async def example_usage():
     """Example usage of the MCP Debug Tool"""
     async with MCPDebugTool(
-        gcp_endpoint="https://your-gcp-api",
-        auth_token="your-oauth-token"
+        gcp_endpoint="https://your-gcp-api", auth_token="your-oauth-token"
```

Ruff (0.11.9)

Possible hardcoded password assigned to argument: "auth_token"

(S106)